Project.toml

```diff
@@ -49,7 +49,7 @@ StridedViews = "0.3, 0.4"
 Test = "1"
 TupleTools = "1.6"
 VectorInterface = "0.4.1,0.5"
-cuTENSOR = "2.1.1"
```

Comment:
I know it doesn't matter too much for this PR, but I think this minimal version was really required, mostly because we are relying on this PR : https://github.com/JuliaGPU/CUDA.jl/pull/2356